src/java-runtime/index.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import findJavaHome = require("find-java-home");
+
+export async function validateJavaRuntime(): Promise<boolean> {
```

Comment:
This part of logic is already in language server. Should we avoid duplicate?

Reply:
The purpose of this change is to make the info very visible and intuitive.